src/browser-lib/single-spa-angular.ts

```diff
@@ -91,7 +91,7 @@ function mount(opts, props) {
         opts.bootstrappedNgZone = module.injector.get(opts.NgZone)
         opts.bootstrappedNgZone._inner._properties[opts.zoneIdentifier] = true;
         window.addEventListener('single-spa:routing-event', opts.routingEventListener)
-        
+
```

Comment:
Please revert whitespace changes to files that you didn't change.

Reply:
My vscode/tslint do this all the time --', sure